main.go

```diff
@@ -136,6 +137,26 @@ func (c *collectorSet) Type() string {
 	return "string"
 }
 
+type namespaceList []string
+
+func (n *namespaceList) String() string {
+	return strings.Join(*n, ",")
+}
+
+func (n *namespaceList) IsAllNamespaces() bool {
+	return len(*n) == 1 && (*n)[0] == metav1.NamespaceAll
+}
+
+func (n *namespaceList) Set(value string) error {
+	splittedNamespaces := strings.Split(value, ",")
+	*n = append(*n, splittedNamespaces...)
+	return nil
+}
+
+func (n *namespaceList) Type() string {
+	return "string"
```

Comment:
This would be "namespaceList"

Reply:
Fixed in 239e9f2. I will squash after review.